data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/request/query/QueryRegistrationRequestsMGM.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "QueryRegistrationRequestsMGM",
+  "namespace": "net.corda.data.membership.db.request.query",
+  "doc": "MGM query for registration requests.",
+  "fields": [
+    {
+      "name": "requestingMemberX500Name",
```

Comment:
Is this different to the requesting member in `MembershipRequestContext`?

Reply:
(if it is maybe a different name could be better to avoid confusion)

Reply:
Renamed to `requestSubjectX500Name`